test/export/ExtensionsExporter.test.ts

```diff
@@ -0,0 +1,66 @@
+import { ExtensionExporter } from '../../src/export';
+import { FSHTank, FSHDocument } from '../../src/import';
+import { FHIRDefinitions, load } from '../../src/fhirdefs';
+import { Extension } from '../../src/fshtypes';
+
+describe('ExtensionsExporter', () => {
```

Comment:
For consistency, rename the file and this line to `ExtensionExporter` rather than `ExtensionsExporter`.